src/main/java/com/ubc/cpsc304/hotelify/controller/dto/AuthenticationResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package com.ubc.cpsc304.hotelify.controller.dto;
+
+import lombok.Data;
+
+/**
+ * ResponseDto for authentication purposes
+ * Created by Rex on 2018-11-05
+ */
+@Data
+public class AuthenticationResponseDto {
```

Comment:
Since there is only one field, use `AllArgsConstructor` so that you dont have to set it.

Reply:
done